### PR TITLE
update to work on windows

### PR DESCRIPTION
**Pull request by yuenmichelle1**

### CODE REVIEW

**Comment by yuenmichelle1** [comment]
This was a deceivingly tricky one so I wanted to walk through steps:
When running through windows, .sh scripts had empty lines that did not play well in a windows machine. 

After removing blank lines, .sh scripts were still making windows unhappy, after a quick stack overflow, i had to check for weird "unseen" characters by running 
`od -c start_server.sh`, realizing there were still lingering `\r`s and then within a terminal of a working container installed `dos2unix` by doing `apt-get update` then `apt-get install dos2unix` then once installed converted .sh files by running `dos2unix start_server.sh`

**TL; DR VERSION** ran through these steps

- remove blank lines from `start_server.sh` and `start_worker.sh`
- ran `od -c start_server.sh` and `od -c start_worker.sh` to find any funny characters 
- installed dos2unix by running `apt-get update` then `apt-get install dos2unix`
- converted files with `dos2unix start_server.sh` and `dos2unix start_worker.sh`

-
